test/permissions_client/core_test.clj

```diff
@@ -103,8 +103,7 @@
 (deftest test-delete-subject
   (let [subject-id "54430509-794e-497e-8ccd-ba01c713ef9f"]
     (with-fake-routes {(fake-url "subjects" subject-id) {:delete (delete-subject-response-fn subject-id)}}
-      (pc/delete-subject (create-fake-client) subject-id)
-      (is true "Subject deleted successfully."))))
```

Comment:
Eastwood was complaining about the assertion that always passes, which was here because an exception will be thrown if something goes wrong. I decided that these messages probably weren't really necessary anyway, so I removed the assertions.